index.js

```diff
@@ -816,6 +816,8 @@ const runTests = async () => {
   debug(`Cypress options ${JSON.stringify(cypressOptions)}`)
 
   const onTestsFinished = (testResults) => {
+    core.setOutput('runResults', testResults)
```

Comment:
Discussed with the team. For simplicity, we would like to scope the exposed runResults stringified object to `version`, `totalDuration`, totalFailed`, `totalPassed`, `totalPending`, `totalSkipped`, `totalSuites` and `totalTests`. The results object can get quite large depending on the size of your test suite and if more information is needed, it can be parsed and save in the [`afterRun` hook](https://docs.cypress.io/api/plugins/after-run-api).

Reply:
@emilyrohrbough can logging the entire results object be an optional input parameter instead? e.g.
```yaml
output-entire-results-object: true 
```
This way, by default, a summary is output but still leaves us the option to get the entire object. This way one does not need to go in and change all the test files to log an object (only needs a version update and addition of a parameter)
CC @mihaisee